### DISCUSSION TITLE
PCM5100A DAC support?

### DISCUSSION THREAD

**Discussion by mungewell** [probonopd/MiniDexed#184] [Q&A]
Front page mentions PCM5102A and PCM5122, but do you also support PCM5100A? Or is it not possible?

Local vendor sells 'DAC + button + display' sheilds with line or headphone level output:
https://www.pishop.ca/product/pirate-audio-line-out-for-raspberry-pi/

I'm not expecting the display to work, not at least for a while ;-)

**Comment by probonopd** [comment]
Hello @mungewell, PCM5100A _should_ be compatible to the PCM5102A but I have not tested it due to the lack of hardware.
